lib/_http_outgoing.js

```diff
@@ -97,7 +97,7 @@ function OutgoingMessage() {
 
   this.finished = false;
   this._headerSent = false;
-  this[kIsCorked] = false;
+  this[ kIsCorked ] = false;
```

Comment:
format: style mismatch:
```suggestion
  this[kIsCorked] = false;
```

`eb1a3af9` ␣ at column 8 should be removed.
`cc749a7d` ␣ at column 18 should be removed.